### DISCUSSION TITLE
Homebrew install previous version

### DISCUSSION THREAD

**Discussion by juan-conde** [libvips/libvips#3207] [Q&A] [answered]
Hi! 
Im having an issue installing vips via homebrew. When running brew install vips, version 8.13.3 is downlaoded, with no way of specifying which version i need to download.
Im actually needing 8.13.2. How can i do this?
Thank you!

**Comment by jcupitt** [comment, marked as answer]
Hi @juan-conde,

Unfortunately, homebrew doesn't support downgrading packages. You'll need to download and build 8.13.2 yourself.

It's pretty easy: just get the `vips.rb` for 8.13.2, drop that into your homebrew formula folder, and remove and reinstall vips.

**Reply by piloudefoug** [reply]
Hello @jcupitt , could you tell me where do you get the vips.rb ?

**Reply by jcupitt** [reply]
It's in the homebrew-core repo:

https://github.com/Homebrew/homebrew-core/blob/master/Formula/v/vips.rb

Step back in time to 8.13.2 and get that version.
